sdk/servicebus/service-bus/src/receivers/sessionReceiver.ts

```diff
@@ -354,7 +358,7 @@ export class ServiceBusSessionReceiverImpl implements ServiceBusSessionReceiver
           .peekMessagesBySession(
             this.sessionId,
             maxMessageCount,
-            options.omitMessageBody,
+            omitMessageBody,
```

Comment:
No sample?
Can we put a readme snippet under `## Experimental` section to set a precedence for all such features that would follow in the future?

Reply:
I had a sample but removed because this feature is only for internal usage. We can wait until we have an experimental feature that is intended for public